mantidproject/mantid
# Reflectometry Preview tab Edit ROI

**Description of work.**
This PR makes it possible to edit multiple different rectangle ROI's when the toolbar Edit button is selected. It also connects up the toolbar to give a visual indication for which tool you have selected.

**To test:**
1. Open ISIS Reflectometry interface
2. Load `INTER45455_inst`, and change angle to 1.00
3. Mess around with the toolbar and try edit/draw regions of interests. Make sure the behaviour is the same as seen on the inst view.

Refs #34174

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
